moonscript/stalin-sort.moon

```diff
@@ -0,0 +1,16 @@
+-- based on the python implementation
+stalinSort = (l) ->
+  max    = l[1]
+  filter = (x) ->
+    max = x
+    x
+  [filter elem for elem in *l when elem >= max]
+
+test = (l) ->
```

Comment:
Just for pattern matters, could you remove the testing and put an exportation, please.